ui/src/app/components/take-demographic-survey-v2.tsx

```diff
@@ -99,7 +95,10 @@ export const TakeDemographicSurveyV2BannerMaybe = () => {
   // Users will continue to see the banner, until they have taken the survey
   // If the time is up, update the text message rather than hide the banner
   if (timeLeftDisplayStr === '') {
-    notificationText += ' This will only take few minutes';
+    notificationText +=
+      'Please complete the updated Researcher Demographic Survey ' +
```

Comment:
needs a . here

Reply:
Updated, however the text will change as part of the story https://precisionmedicineinitiative.atlassian.net/browse/RW-8644